README.md

```diff
@@ -44,3 +44,82 @@ along with this program.  If not, see https://www.gnu.org/licenses/
 ___
 
 Project Website: https://metrocs.github.io/imagelab/
+
+## Build Instructions 
+
+BUILD INSTRUCTIONS
+
+◾️Introduction
+
+This documents intent is to support developers in understanding how to 
+build the Image Lab project. This document offers support on key useful commands and error solutions that can make building Image Lab a smoother process.
+
+◾️Before You Build
+- Install Gradle
```

Comment:
There should be some explanation of _why_ before instructing the user to install something on their system.
Include a link to the external product's website.